docs/json_data/sysfs_attributes_drm.json

```
@@ -539,6 +539,138 @@
             "units": "none",
             "writeable": false,
             "alias": ""
+        },
+        "ACT_FREQ": {
+            "attribute": "freq0/act_freq",
+            "scalar": 1e6,
+            "description": "Actual GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "CUR_FREQ": {
+            "attribute": "freq0/cur_freq",
+            "scalar": 1e6,
+            "description": "Current GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_STATUS"
+        },
+        "MAX_FREQ": {
+            "attribute": "freq0/max_freq",
+            "scalar": 1e6,
+            "description": "Maximum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MAX_CONTROL"
+        },
+        "MIN_FREQ": {
+            "attribute": "freq0/min_freq",
+            "scalar": 1e6,
+            "description": "Minimum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MIN_CONTROL"
+        },
+        "RP0_FREQ": {
+            "attribute": "freq0/rp0_freq",
+            "scalar": 1e6,
+            "description": "Maximum non-overclocked GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_MAX_AVAIL"
+        },
+        "RPA_FREQ": {
+            "attribute": "freq0/rpa_freq",
+            "scalar": 1e6,
+            "description": "Actual GPU RPA frequency",
```

```suggestion
            "description": "Achievable GPU frequency",
```
Does the `a` in RPa mean achievable?